CONTRIBUTING.md

```diff
@@ -336,3 +361,31 @@ The Rule of Zero states that you do not provide any of the functions that the co
 The goal is to let the compiler provide optimal versions that are automatically maintained when more member variables are added.
 
 The [original article](http://flamingdangerzone.com/cxx11/rule-of-zero/) provides the background, while a [follow up article](https://turingtester.wordpress.com/2015/06/27/cs-rule-of-zero/) explains techniques for implementing nearly 100% of the time.
+
+
+Collaboration Guide
+-------------------
+The basic steps to starting to work on a change and submitting it:
+
+1. Fork the repo
+2. Clone your fork
+3. Add the main repo as a remote (e.g. you can name it upstream)
+3. `git checkout -b branch_name` (make sure to use a descriptive name for your branch)
```

Comment:
Also, two `3.`s even though it doesn't show up like that on the actual file.... weird?